### PR TITLE
New package: python-ansicolor-0.2.4

### PR DESCRIPTION
**Pull request by lafrenierejm**
This library is needed for [vint](https://github.com/Kuniwak/vint), a VimL linter that I intend to package shortly.

### CODE REVIEW

File: srcpkgs/python-ansicolor/template [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	vlicense LICENSE
+}
+
+python3-ansicolor_package() {
+	noarch=yes
+	pycompile_module="ansicolor"
+	short_desc="${short_desc/Python2/Python3}"
+	pkg_install() {
+		vmove usr/lib/python3*
+		vlicense LICENSE
```
Comment (Vaelatern) [inline srcpkgs/python-ansicolor/template]: Not needed for the subpackage

Comment (lafrenierejm) [inline srcpkgs/python-ansicolor/template]: Why are those lines not needed for the Python3 subpackage when they are needed for e.g., python-yamllint?

Comment (asergi) [inline srcpkgs/python-ansicolor/template]: @Vaelatern got wrong. Anyway, the `Apache-2.0` license is part of `base-files`, so there's no need to install the license at all.

Comment (Vaelatern) [inline srcpkgs/python-ansicolor/template]: Except it does get personalized at the end....

Comment (lafrenierejm) [inline srcpkgs/python-ansicolor/template]: Apache does require customizing the license:

> To apply the Apache License to your work, attach the following boilerplate notice, with the fields enclosed by brackets "[]" replaced with your own identifying information.

That is taken from [https://www.apache.org/licenses/LICENSE-2.0.txt](https://www.apache.org/licenses/LICENSE-2.0.txt).

Comment (asergi) [inline srcpkgs/python-ansicolor/template]: Remove `vlicense`.

File: srcpkgs/python-ansicolor/template [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Template file for 'python-ansicolor'
+pkgname=python-ansicolor
+version=0.2.4
+revision=1
+noarch=yes
+wrksrc="ansicolor-${version}"
+build_style=python-module
+pycompile_module="ansicolor"
+hostmakedepends="python-setuptools python3-setuptools"
+makedepends="python-devel python3-devel"
```
Comment (asergi) [inline srcpkgs/python-ansicolor/template]: `depends=python`. You can remove the `makedepends` line too.

File: srcpkgs/python-ansicolor/template [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+license="Apache-2.0"
+distfiles="${homepage}/archive/${version}.tar.gz"
+checksum=f66c4e3446d419813de1bfd011f6dab3d96de9369388727fed434a63e7d8edbd
+
+post_install() {
+	vlicense LICENSE
+}
+
+python3-ansicolor_package() {
+	noarch=yes
```
Comment (asergi) [inline srcpkgs/python-ansicolor/template]: `depends="python3"`

**Comment by lafrenierejm** [comment]
Changes have been made, @asergi.

**Pull request merged**